sdk/clientcore/core/src/test/java/io/clientcore/core/http/client/HttpClientProvidersIT.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.core.http.client;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledForJreRange;
+import org.junit.jupiter.api.condition.JRE;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+/**
+ * Tests for the {@link HttpClient} class.
+ * <p>
+ * Now that the default HttpClient, and related code, are using multi-release JARs this must be an integration test as
+ * the full JAR must be available to use the multi-release code.
+ */
+@DisabledForJreRange(max = JRE.JAVA_11)
+public class HttpClientProvidersIT {
+    @Test
```

Comment:
Just curious, so we won't do any pipeline + client (Integration) testing for Jav [8,11)? Or are planning to add that in future PRs?

Reply:
We won't be testing the default HttpClient in Java 8 or 11 but this will be tested in Java 17 and 21. There is no plan on having a default HttpClient in Java 8 and 11 for now.